static/title-bar.less

```diff
@@ -0,0 +1,54 @@
+@import "./variables/ui-variables";
+
+.title-bar {
+  height: 23px;
+
+  flex-shrink: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  font-family: -apple-system, BlinkMacSystemFont, "Helvetica Neue";
+  font-size: @title-bar-text-size;
+  color: @title-bar-text-focused;
+  -webkit-user-select: none;
+
+  padding: 0 10px 0 70px;
+  overflow: hidden;
+
+  .title {
+    flex: 0 1 auto;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
+
+  background-image: linear-gradient(@title-bar-gradient-focused);
+  border-bottom: 1px solid @title-bar-border-color-focused;
+
+  .is-blurred & {
+    color: @title-bar-text-blurred;
+    background-image: linear-gradient(@title-bar-gradient-blurred);
+    border-bottom-color: @title-bar-border-color-blurred;
+  }
+
+  [title-bar-style="combined"] & {
+    display: none;
+    position: absolute;
+  }
+}
+
+[title-bar-style="combined"] .tab-bar {
```

Comment:
Oh dang, these ended up in there by accident. You guessed correctly, I was already working on a compact/combined (`hidden` and `hidden-inset`) titlebar.
I'll remove them for now, since the dragging fix you mentioned is blocking any serious attempts at this.